docs/reference/commandline/run.md

```diff
@@ -272,6 +296,12 @@ will automatically create this directory on the host for you. In the
 example above, Docker will create the `/doesnt/exist`
 folder before starting your container.
 
+#### Use a read-only bind mount
+
+```bash
+$ docker run --mount src=/icanwrite,dst=/icanwrite,readonly busybox touch /icanwrite/here
```

Comment:
Looks like we're mixing two things in the section; the existing example was about a read-only container, and using an (anonymous) volume to make parts of the container writable